src/storage/conversion/ConstantConverter.ts

```diff
@@ -19,26 +38,41 @@ import type { RepresentationConverterArgs } from './RepresentationConverter';
 export class ConstantConverter extends RepresentationConverter {
   private readonly filePath: string;
   private readonly contentType: string;
+  private readonly options: ConstantConverterOptions;
 
   /**
    * Creates a new constant converter.
    *
    * @param filePath - The path to the constant representation.
    * @param contentType - The content type of the constant representation.
+   * @param options - Extra options for the converter.
    */
-  public constructor(filePath: string, contentType: string) {
+  public constructor(filePath: string, contentType: string, options: ConstantConverterOptions) {
```

Comment:
```suggestion
  public constructor(filePath: string, contentType: string, options: ConstantConverterOptions = {}) {
```

Reply:
You caught me, I had this first, but then I needed to add a new unit test and I didn't want to bother since there is no situation where you would ever want this class with an empty object anyway 😄 (because that would imply conversion can never happen)

Reply:
Or maybe go for most permissive. Just that the class does something when you instantiate it without options.